### DISCUSSION TITLE
Require `vike-{react,vue,solid}` or add a warning?

### DISCUSSION THREAD

**Discussion by brillout** [vikejs/bati#263] [Misc]
I've added a warning telling users to only go for a custom integration (instead of using `vike-{react,vue,solid}`) if they have a clear reason not to. For example:
 - https://vike.dev/new#without-vike-react-vue-solid
 - https://vike.dev/solid#manual-integration

Maybe we can also add that warning to Bati. A user scaffolded a Vike app without `vike-{react,vue,solid}` using Bati and got confused.

Also, I think we can even require `vike-{react,vue,solid}`, especially if it makes Bati easier to maintain.

**Comment by magne4000** [comment]
That's a good point, using Bâti without those integrations is mostly pointless anyway.

**Comment by magne4000** [comment]
Fixed by #390
